indico/modules/events/editing/client/js/components/FileManager/index.jsx

```diff
@@ -104,13 +104,17 @@ FileType.defaultProps = {
   uploads: {},
 };
 
-export default function FileManager({downloadURL, uploadURL, fileTypes, files}) {
+export default function FileManager({onChange, downloadURL, uploadURL, fileTypes, files}) {
   const _fileTypes = useMemo(() => mapFileTypes(fileTypes, files), [fileTypes, files]);
   const [state, dispatch] = useReducer(reducer, {
     fileTypes: _fileTypes,
     uploads: {},
   });
 
+  useEffect(() => {
```

Comment:
using an effect for this seems to be a bit strange. I'd rather call onChange explicitly after an action affecting the final list of file types happened (e.g. the user clicking on delete, an upload finishing, etc.)